src/bindings/js/node/include/model_wrap.hpp

```diff
@@ -84,6 +84,13 @@ class ModelWrap : public Napi::ObjectWrap<ModelWrap> {
      */
     Napi::Value is_dynamic(const Napi::CallbackInfo& info);
 
+    /**
+     * @brief Helper function to access model outputs shape.
+     * @param info Contains information about the environment and passed arguments
+     * @return A Javascript Array containing Arrays of output shapes
```

Comment:
It returns shape. Shape represents in JS as array of numbers. It doesn't match with this description.